plugins/reporters/static-html/src/main/kotlin/TablesReportModelMapper.kt

```diff
@@ -39,8 +39,6 @@ import org.ossreviewtoolkit.reporter.ReporterInput
  */
 internal object TablesReportModelMapper {
     fun map(input: ReporterInput): TablesReport {
-        // TODO: Use the prefixes up until the first '.' (which below get discarded) for some visual grouping in the
-        //       report.
         val labels = input.ortResult.labels.mapKeys { it.key.substringAfter(".") }
```

Comment:
The commit message should probably say *why* the `TODO` is obsolete... is it simply not planned anymore to implement the mentioned functionality? Or not possible / meaningful anymore due to the recent refactorings?